src/Routers/FilesRouter.js

```diff
@@ -299,11 +298,9 @@ export class FilesRouter {
       const { filesController } = config;
       const { filename } = req.params;
       const data = await filesController.getMetadata(filename);
-      res.status(200);
-      res.json(data);
+      res.status(200).json(data);
```

Comment:
Please revert all changes that are unnecessary for the express upgrade, like these style changes.